ansible/roles/couchdb/templates/couchdb.init.j2

```diff
@@ -88,6 +88,9 @@ start_couchdb () {
         chown $COUCHDB_USER "$RUN_DIR"
     fi
     command="$COUCHDB -b"
+    if test -n "$ENV_ERL_FLAGS"; then
+        command="env ERL_FLAGS=\"$ENV_ERL_FLAGS\" $command"
+    fi
```

Comment:
the actual command gets run with su, which cleans the environment variables before running the command. so it had to be added explicitly to the env variables for the command. i renamed it to `ENV_ERL_FLAGS` to avoid confusion